libraries/eosiolib/ignore.hpp

```diff
@@ -0,0 +1,25 @@
+#pragma once
+
+namespace eosio {
+   /**
+    * Tells the datastream not ignore this type, but allows the abi generator to add the correct type.
+    * @brief Tells the datastream not ignore this type, but allows the abi generator to add the correct type.
```

Comment:
"Tells the datastream _to_ ignore this type, ..." for this line and the one above.

I think it is worth also putting in the comments here a statement how an `ignore` currently should not precede any non-`ignore` arguments in the action function signature and also: how that constraint is enforced by the CDT compiler and how it will be relaxed in the future.